nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -765,6 +764,20 @@ define([
             updateEditingState();
         }
 
+        /**
+         * Reset the cell to the editing state
+         */
+        function doResetCellAction() {
```

Comment:
quick and dirty reset action -- I just added this to make UI-based testing easier. It needs to be checked to ensure all the appropriate resetting is accomplished.

Reply:
Yeah. That's a TODO. Should we make a ticket?